Documentation/CRDs/Block-Storage/ceph-block-pool-rados-namespace-crd.md

```diff
@@ -81,3 +81,42 @@ parameters:
   pool: replicapool
   ...
 ```
+
+### Mirroring
+
+#### Prerequisites
+
+Enable the mirroring in the default rados namespace(CephBlockPool), refer [CephBlockPool documentation](ceph-block-pool-crd.md#mirroring).
+
+Configure the rados namespace CRD with mirroring spec
+
+```yaml
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPoolRadosNamespace
+metadata:
+  name: namespace-a
+  namespace: rook-ceph # namespace:cluster
+spec:
+  # The name of the CephBlockPool CR where the namespace is created.
+  blockPoolName: replicapool
+  mirroring:
+    enabled: false
+    mode: image
+    remoteNamespace: "test"
+    snapshotSchedules:
+```
+
+#### Mirroring Spec
+
+- `mirroring`: Sets up mirroring of the rados namespace
+    - `enabled`: whether mirroring is enabled on that rados namespace (default: false)
+    - `mode`: mirroring mode to run, possible values are "pool" or "image" (required). Refer to the [mirroring modes Ceph documentation](https://docs.ceph.com/docs/master/rbd/rbd-mirroring/#enable-mirroring) for more details
+    - `remoteNamespace`: Name of the rados namespace on the peer cluster where the namespace should get mirrored.
```

Comment:
We can default to the same namespace, right?
```suggestion
    - `remoteNamespace`: Name of the rados namespace on the peer cluster where the namespace should get mirrored. The default is the same rados namespace.
```

Reply:
In that case we dont need to use `remotenamesapce`, we don't have default as we want it to be different

Reply:
Per separate thread, Rook's default would be `null`, which means the param is not passed, and then Ceph's default will be the same namespace. So my suggestion still seems valid.